packages/flutter/lib/src/animation/animations.dart

```diff
@@ -49,7 +60,9 @@ class AlwaysStoppedAnimation extends Animation<double> {
   AnimationDirection get direction => AnimationDirection.forward;
 }
 
+/// A mixin to help with building animations that proxy the value of another animation.
 abstract class ProxyAnimatedMixin {
```

Comment:
ProxyAnimatedMixin is confusingly similar to ProxyAnimation, given that they have no relationship to each other. I don't have a better name suggestion though.